### ISSUE TITLE
loop data missing

### ISSUE THREAD

**Issue by davidmlb** [dabear/NightscoutShareServer#8]
thanks a lot for the Nightscoutshare Sever - I  deply by azure, and i load the value and I can see!!!
![image](https://user-images.githubusercontent.com/28386261/55703000-ae160f00-59d8-11e9-9718-975fa90b6075.png), then i Modify the loop with the url and then I can select the cgm source in loop as a Custom, but the glucose data in not loading in loop. Any idea I'm doing wrong? i tried it in loop 1.9.2 and 1.9.3 
thanks a loot

**Comment by benfiscal** [comment]
Have you registered your pump and RileyLink? It needs to be done first, otherwise the CGM values aren’t fetched.

**Comment by dabear** [comment]
If nightscoutshareserver is working as intended, but you have problems with loop, then this question is out of scope for this project :)
